### PR TITLE
Positive real balanced truncation (PRBT)

### PR DESCRIPTION
**Pull request by Jonas-Nicodemus**
This PR implements PRBT for passive LTIModels.

### CODE REVIEW

File: src/pymor/models/iosys.py [hunk 1]
```diff
@@ -789,15 +791,21 @@ def _gramian(self, typ, mu=None):
         elif typ == 'lqg_o_lrcf':
             return solve_ricc_lrcf(A, E, B.as_range_array(mu=mu), C.as_source_array(mu=mu),
                                    trans=True, options=options_ricc_lrcf)
+        elif typ == 'pr_c_lrcf':
+            return solve_pos_ricc_lrcf(A=A, E=None, B=A.source.zeros(), C=-C.as_source_array(mu=mu),
+                                       R=to_matrix(D + D.H, 'dense'), S=B.H.as_source_array(mu=mu))
+        elif typ == 'pr_o_lrcf':
+            return solve_pos_ricc_lrcf(A=A, E=None, B=-B.as_range_array(mu=mu), C=A.source.zeros(),
```
Comment (steff-mueller) [inline src/pymor/models/iosys.py:798]: Is it intentional that we do not pass the `E` from the model, but set `E=None`?

File: src/pymor/models/iosys.py [hunk 2]
```diff
@@ -789,15 +791,21 @@ def _gramian(self, typ, mu=None):
         elif typ == 'lqg_o_lrcf':
             return solve_ricc_lrcf(A, E, B.as_range_array(mu=mu), C.as_source_array(mu=mu),
                                    trans=True, options=options_ricc_lrcf)
+        elif typ == 'pr_c_lrcf':
```
Comment (steff-mueller) [inline src/pymor/models/iosys.py:794]: I think we should mention in the docstring for the `pr_c_lrcf` and `pr_o_lrcf` Gramians that $D+D^T$ must be invertible, and maybe add an assertion like
```python
assert not isinstance(D, ZeroOperator), "D+D^T must be invertible."
```
Alternatively, we might opt to automatically add a perturbation similar to https://github.com/pymor/pymor/blob/dfe7c32dbb4b92b8128c74837cb65a13e248d970/src/pymordemos/phlti.py#L131, and maybe log a warning.

**Comment by sdrave** [comment]
@Jonas-Nicodemus, you should add yourself to `AUTHORS.md` ...

File: src/pymor/models/iosys.py (hunk 2 above)
Comment (pmli) [reply src/pymor/models/iosys.py:960]: For now, I would just add a note in the documentation.

**Review by pmli — APPROVED** [review]

**Pull request merged**